astropy_helpers/openmp_helpers.py

```diff
@@ -0,0 +1,133 @@
+# This module defines functions that can be used to check whether OpenMP is
+# available and if so what flags to use. To use this, import the
+# add_openmp_flags_if_available function in a setup_package.py file where you
+# are defining your extensions:
+#
+#     from astropy_helpers.openmp_helpers import add_openmp_flags_if_available
+#
+# then call it with a single extension as the only argument:
+#
+#     add_openmp_flags_if_available(extension)
+#
+# this will add the OpenMP flags if available.
+#
+# This file includes code heavily adapted from astroscrappy, originally released
+# under the following license:
```

Comment:
Maybe this should go under the ``licenses`` directory in the root of the helpers instead of embedded in this file?

Reply:
Done